packages/flutter/lib/src/widgets/text.dart

```diff
@@ -1401,3 +1532,22 @@ class _SelectableTextContainerDelegate extends MultiSelectableSelectionContainer
     super.didChangeSelectables();
   }
 }
+
+class _TextSpanContentRange extends SelectedContentRange<TextSpan> {
```

Comment:
I left another comment somewhere wondering about subclassing SelectedContentRange, and the start vs. startOffset parameters, and whether it is all necessary or not. Just pointing it out again here.

Reply:
`SelectedContentRange` is now concrete, and I removed the subclasses. Thanks for the suggestion!